src/components/Scrollbar.tsx

```diff
@@ -13,18 +25,34 @@ interface ScrollbarProps {
   fade?: FadeType
 }
 
+const scrollContainer = classnames(
+  display('flex'),
+  flexDirection('flex-col'),
+  flexGrow('grow'),
+  position('relative')
+)
+
 export default function ({
   children,
   maxHeight = 350,
```

Comment:
Rename this prop to `initHeight` or something like this, it doesn't represent the actual `maxHeight`